### DISCUSSION TITLE
Sobre cantidad máxima de ordenes en un heap (parte 1)

### DISCUSSION THREAD

**Discussion by ManuelV03** [IIC2133-PUC/2024-1#75] [Tarea 1] [answered]
Hola, me entró una duda en la parte 1 acerca de si hay un máximo de ordenes que pueden entrar en las colas de prioridad del programa. Considero que saber esto es importante porque la forma más usual de implementar un min/max heap es a través de un array, por lo que cuando se solicita memoria al programa para el array de la cola, es necesario saber el largo máximo que este puede tener, por lo que si no sabemos el máximo de ordenes (o por lo menos una cota superior), tendríamos que construir el heap de otra manera.

**Comment by 4n1t4** [comment, marked as answer]
Holi!
Una posibilidad es aproximar al peor caso y utilizar una capacidad del tamaño de N_inputs.

**Reply by ManuelV03** [reply]
Cierto, no me había fijado que estaba ese N_inputs.
Muchas gracias!!
